server/xpub-model/entities/user/helpers/elife-api.js

```diff
@@ -46,16 +46,19 @@ const convertPerson = apiPerson => {
 
 const people = async role => {
   logger.debug('Fetching editors from /people', { role })
+
   let items = []
   let response
   let page = 1
   do {
-    response = await request('people', {
-      order: 'asc',
-      page,
-      'per-page': 100,
-      type: role,
-    })
+    let query = `order=asc&page=${page}&per-page=100`
+    if (role) {
+      role.split(',').forEach(r => {
+        query += `&type[]=${r}`
```

Comment:
We can use `qs` to handle query string arrays. [Here's an example.](https://github.com/visionmedia/superagent/issues/629#issuecomment-109197779)

Reply:
That is not using the PHP syntax :( ... like `key[]=value1&key[]=value2`

Reply:
Ah, that's just because it's encoded. You can set `encode` to false:

```javascript
qs.stringify(
  query, 
  { arrayFormat: 'brackets', encode: false }
)
```

Reply:
Have discussed this with @diversemix it might not be worth adding a new external library for this one implementation. It doesn't add too much extra code and as we're not having to do this conversion anywhere else in the app yet it doesn't seem necessary to make this conversion process an abstract function. I would leave as is unless we need to do this elsewhere in the future.